spec/jasmine/build/bundle.js

```diff
@@ -2,7 +2,7 @@
 module.exports=[
```

Comment:
### Critical

@andela-rbabalola You should `NEVER` include bundled files like this in version control, it can be generated from source, so it's redundant. It's the same reason you never add `node_modules/` in production.